docs/middleware/logger.md

```
@@ -108,6 +129,7 @@ Writing to os.File is goroutine-safe, but if you are using a custom Output that
 | TimeZone         | `string`                   | TimeZone can be specified, such as "UTC" and "America/New_York" and "Asia/Chongqing", etc                                        | `"Local"`                                                             |
 | TimeInterval     | `time.Duration`            | TimeInterval is the delay before the timestamp is updated.                                                                       | `500 * time.Millisecond`                                              |
 | Output           | `io.Writer`                | Output is a writer where logs are written.                                                                                       | `os.Stdout`                                                           |
+| LoggerFunc           | `func(c fiber.Ctx, data *Data, cfg Config) error`  | You can use custom loggers with Fiber by using this field. This field is really useful if you're using Zerolog, Zap, Logrus, apex/log etc. If you don't define anything for this field, it'll use the default logger of Fiber.  | `see default_logger.go defaultLoggerInstance`                                                           |                                           |
```

Markdownlint (0.35.0)

Expected: 4; Actual: 5; Too many cells, extra data will be missing
Table column count

(MD056, table-column-count)